### ISSUE TITLE
Why doesn't the header node use overflow-y: scroll?

### ISSUE THREAD

**Issue by mike1a2a3a** [SitePen/dgrid#1273]
Right now dgrid seems to calculate the width of a scrollbar so it can set the 'right' value of the header node.  This allows it to line up with the content node below it.  Why not just make it right:0 and add overflow-y: scroll on the header node, then perhaps cover the scrollbar that appears with the small white corner?  
This will allow the header node to be perfectly aligned with the content body, even when you zoom in.

**Comment by kfranqueiro** [comment]
That's a good point about the discrepancy caused by zooming.  Wouldn't we still run into that issue with the covering node if we took the approach of leaving a scrollbar there and then covering it, though?

**Comment by mike1a2a3a** [comment]
Yeah it might bleed in one or two pixels while zooming but atleast the columns will be aligned

**Comment by kfranqueiro** [comment]
Oh, so that means this is actually the cause of the misalignment you mentioned in #1272.

**Comment by mike1a2a3a** [comment]
No, #1272 is entirely unrelated.  In that issue I'm wondering why each row is defined within a separate table.  I can't think of a good reason why its like this, and I can think of a lot of benefits to changing it to a single-table approach.  
Some benefits:
- You could change the table-layout to be 'auto' instead of 'fixed' for certain grids to make them size based on the cell's content !
- Much less dom elements
- No more column alignment issues
- No more dynamically-generated CSS stylesheet rules, you can simply set the width attribute on the table's <colgroup> tag.
I know kendo UI's grid uses a single table, you can see it here: http://demos.telerik.com/kendo-ui/grid/column-resizing

**Comment by kfranqueiro** [comment]
I know the question itself is unrelated (and there are reasons why it is the way it is, I'll explain when I have a chance), but you mentioned misalignment there that seems to be the same thing you alluded to with "at least the columns will be aligned" here in your previous comment.
